cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/sponge/platform/SpongeClientServerPingEventCaller.java

```diff
@@ -0,0 +1,79 @@
+package de.dytanic.cloudnet.ext.bridge.sponge.platform;
+
+import de.dytanic.cloudnet.ext.bridge.BridgeHelper;
+import de.dytanic.cloudnet.ext.bridge.server.BridgeServerHelper;
+import org.jetbrains.annotations.NotNull;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.event.SpongeEventFactory;
+import org.spongepowered.api.event.cause.Cause;
+import org.spongepowered.api.event.cause.EventContext;
+import org.spongepowered.api.event.server.ClientPingServerEvent;
+import org.spongepowered.api.network.status.StatusResponse;
+import org.spongepowered.api.scheduler.SpongeExecutorService;
+
+import java.util.concurrent.TimeUnit;
+
+public final class SpongeClientServerPingEventCaller {
+
+    private SpongeClientServerPingEventCaller() {
+        throw new UnsupportedOperationException();
+    }
+
+    private static SpongeExecutorService executorService;
+
+    public static void startCalling(@NotNull Object plugin) {
```

Comment:
Codacy found an issue: [The method 'startCalling(Object)' has an NPath complexity of 240, current threshold is 200](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=5823302)